### PR TITLE
Uplift arm64 universal build (uplift to 1.19.x)

### PR DESCRIPTION
**Pull request by bridiver**
Uplift of #7366, #7475, #7482
Fixes https://github.com/brave/brave-browser/issues/13074

Approved, please ensure that before merging: 
- [x] You have checked CI and the builds, lint, and tests all pass or are not related to your PR. 
- [ ] You have tested your change on Nightly. 
- [x] The PR milestones match the branch they are landing to. 

After you merge: 
- [x] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Comment by kjozwiak** [comment]
Restarting `Android` via https://ci.brave.com/job/pr-brave-browser-arm64-1.19.x-android/3/ due to `dist` failing as per https://ci.brave.com/job/pr-brave-browser-arm64-1.19.x-android/2/execution/node/154/log/.

```
23:09:25  TypeError: util.prepareWidevineCdmBuild is not a function
23:09:25      at Object.buildTarget (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-android/src/brave/build/commands/lib/util.js:454:44)
23:09:25      at Command.createDist (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-android/src/brave/build/commands/lib/createDist.js:29:8)
23:09:25      at Command.listener (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-android/src/brave/node_modules/commander/index.js:315:8)
23:09:25      at Command.emit (events.js:314:20)
23:09:25      at Command.parseArgs (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-android/src/brave/node_modules/commander/index.js:654:12)
23:09:25      at Command.parse (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-android/src/brave/node_modules/commander/index.js:474:21)
23:09:25      at Object.<anonymous> (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-android/src/brave/build/commands/scripts/commands.js:211:4)
23:09:25      at Module._compile (internal/modules/cjs/loader.js:999:30)
23:09:25      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
23:09:25      at Module.load (internal/modules/cjs/loader.js:863:32)
```

Restarting `Linux` via https://ci.brave.com/job/pr-brave-browser-arm64-1.19.x-linux/3/ due to `build` failing as per https://ci.brave.com/job/pr-brave-browser-arm64-1.19.x-linux/2/execution/node/159/log/.

```
23:06:57  TypeError: util.prepareWidevineCdmBuild is not a function
23:06:57      at Object.buildTarget (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/brave/build/commands/lib/util.js:454:44)
23:06:57      at Command.build (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/brave/build/commands/lib/build.js:100:10)
23:06:57      at Command.listener (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/brave/node_modules/commander/index.js:315:8)
23:06:57      at Command.emit (events.js:314:20)
23:06:57      at Command.parseArgs (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/brave/node_modules/commander/index.js:654:12)
23:06:57      at Command.parse (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/brave/node_modules/commander/index.js:474:21)
23:06:57      at Object.<anonymous> (/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/brave/build/commands/scripts/commands.js:211:4)
23:06:57      at Module._compile (internal/modules/cjs/loader.js:999:30)
23:06:57      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
23:06:57      at Module.load (internal/modules/cjs/loader.js:863:32)
```

File: build/commands/lib/util.js
Comment (mihaiplesa) [inline build/commands/lib/util.js:519]: removed in the last PR since function was deleted from https://github.com/brave/brave-core/pull/7495/files#diff-6149b8e49614376fc44e8cd42d0ce8661e754d14d3bb4df131c3de178e852b0dL396

**Comment by kjozwiak** [comment]
Looks like `Android` passed 👍 `Linux` failed `build` as per https://ci.brave.com/job/pr-brave-browser-arm64-1.19.x-linux/5/execution/node/159/log/.

```
07:30:39  /home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src
07:30:39  > ninja -C /home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/out/Release brave -k 1
07:30:39  ninja: Entering directory `/home/ubuntu/workspace/pr-brave-browser-arm64-1.19.x-linux/src/out/Release'
07:30:42  ninja: error: '../../third_party/widevine/cdm/linux/x64/widevine_cdm_version.h', needed by 'gen/widevine_cdm_version.h', missing and no known rule to make it
```

File: patches/chrome-installer-mac-BUILD.gn.patch
Comment (mihaiplesa) [inline patches/chrome-installer-mac-BUILD.gn.patch:11]: related
https://github.com/brave/brave-core/blob/master/patches/chrome-installer-mac-BUILD.gn.patch
https://github.com/brave/brave-core/blob/master/patches/chrome-installer-mac-signing-model.py.patch
https://github.com/brave/brave-core/blob/master/patches/chrome-installer-mac-signing-pipeline.py.patch

**Review by mihaiplesa — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.19.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
